contrib/babelfishpg_tsql/sql/babelfishpg_tsql.sql

```diff
@@ -2903,3 +2903,21 @@ END;
 $$
 LANGUAGE 'pltsql';
 GRANT ALL ON PROCEDURE sys.sp_sproc_columns_100 TO PUBLIC;
+
+CREATE OR REPLACE PROCEDURE sys.sp_linkedservers()
+AS $$
+BEGIN
+    SELECT 
+		name AS "SRV_NAME", 
+		provider AS "SRV_PROVIDERNAME", 
```

Comment:
sys.servers. provider is of sys.sysname but we need to return nvarchar(128). Same is true for SRV_PRODUCT